client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -93,6 +98,16 @@ class StatsPostDetail extends Component {
 
 	componentDidMount() {
 		window.scrollTo( 0, 0 );
+
+		const { context } = this.props;
+		recordCurrentScreen( 'postDetails', context.query );
+	}
+
+	componentDidUpdate( prevProps ) {
+		if ( prevProps.statsQueryOptions !== this.props.statsQueryOptions ) {
```

Comment:
`statsQueryOptions` has not been passed to the `stats-post-detail` page, should we use `props.context` for comparison instead?

Reply:
Done in 333c513af08e7e785fd8f7175b7b2f5f4ab0640c